### DISCUSSION TITLE
ARNNConv1D does not take a list[int] as an input for features.

### DISCUSSION THREAD

**Discussion by dashsid** [netket/netket#1410] [Q&A] [answered]
In the NetKet documentation, the object netket.models.ARNNConv1D() is said to take the attribute "features" as a Union[Iterable[int], int]. However, when I take "features" as a list[int], it gives me the following error:

```
TypeError: Failed to hash Flax Module.  The module probably contains unhashable attributes.  Module=ARNNConv1D(
    # attributes
    layers = 4
    features = [5, 10, 20, 80]
    kernel_size = 4
    kernel_dilation = 1
    activation = log_cosh
    use_bias = True
    param_dtype = complex64
    precision = None
    kernel_init = init
    bias_init = zeros
    machine_pow = 2
)
```
I do not get any error when I take "features" as an int. 

Please let me know, how I can give a list of ints as the attribute "features".

**Comment by gpescia** [comment, marked as answer]
Hi @dashsid ,
could you try to use a tuple instead of a list in the model initialization? So [5,10,20,80] -> (5,10,20,80).
I think the error says that you have an unhashable object in your model and since lists are mutable they are not hashable. 

Let me know if this works.

**Reply by dashsid** [reply]
Thank you very much @gpescia. I used a tuple instead of a list and it works for me now!

**Reply by PhilipVinc** [reply]
We should probably update the docstrings
